packages/pwa/app/pages/home/index.jsx

```diff
@@ -131,6 +131,11 @@ Home.propTypes = {
     isLoading: PropTypes.bool
 }
 
-Home.getProps = async () => {}
+Home.getProps = async ({res}) => {
+    // Set the `cache-control` header values to align with the Commerce API settings.
+    if (res) {
+        res.set('Cache-Control', 'public, must-revalidate, max-age=900')
```

Comment:
Why are we using the `must-revalidate` directive here?

<img width="909" alt="Screen Shot 2021-09-14 at 1 53 37 PM" src="https://user-images.githubusercontent.com/28967/133332356-a472b74e-c25b-474f-b43f-c72b5e2a665c.png">

https://pokeinthe.io/2021/09/13/cache-control-recommendations/

This will probably have the desired effect:

```suggestion
        res.set('Cache-Control', 'max-age=900')
```

Reply:
We also may want to have a conversation about whether/how this should be enabled locally to avoid folks doing development and seeing stale content.

Reply:
There are 2 things that we can do here, reply on the developers to disable cache via chrome inspector, or to programatically ignore cache headers when in local development (maybe this is configurable?). There are pro's and cons to both.